src/app/modules/calendar/calendar.component.ts

```diff
@@ -0,0 +1,65 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+import { faCalendarAlt } from "@fortawesome/free-regular-svg-icons";
+
+// Models
+import { CalendarModel } from "./calendar.model";
+
+@Component({
+  selector: "app-calendar",
+  templateUrl: "./calendar.component.html",
+  styleUrls: ["./calendar.component.css"]
+})
+export class CalendarComponent {
+  @Input() calendarModel: CalendarModel;
+  @Output() selectedDates = new EventEmitter<any>();
+
+  showEndDate: boolean = true;
+  valid: boolean = true;
+  errorMessage: string;
+
+  faCalendarAlt = faCalendarAlt;
+
+  constructor() {}
+
+  validateDate(event) {
+    if (event && event.targetElement.name === "date")
+      this.calendarModel.date = event.value;
+    if (event && event.targetElement.name === "endDate")
+      this.calendarModel.endDate = event.value;
```

Comment:
This is really picky, but I think it's against the linting standards to have an if statement without curly braces....